WooCommerce/Classes/ViewRelated/Dashboard/Settings/In-Person Payments/InPersonPaymentsPluginChoicesView.swift

```diff
@@ -0,0 +1,40 @@
+import SwiftUI
+import Yosemite
+
+struct InPersonPaymentsPluginChoicesView: View {
+    @Environment(\.verticalSizeClass) var verticalSizeClass
+
+    var isCompact: Bool {
+        get {
+            verticalSizeClass == .compact
+        }
+    }
+
+    var body: some View {
+        VStack {
+            Text(CardPresentPaymentsPlugins.wcPay.pluginName)
+                .font(.callout)
+                .bold()
+                .padding(.bottom, 1)
+            Text(Localization.conjunctiveOr)
```

Comment:
I'm a bit worried about concatenating strings like that for localization. I don't know of a specific example where this would break, but maybe we could skip the "or", or replace it with a bulleted list to avoid issues?

Reply:
I have the same worry, but also couldn't think of a breaking language.

I like the idea of changing it to a bulleted list too. @adamzelinski - is that ok with you?

Reply:
Changed to bulleted list in https://github.com/woocommerce/woocommerce-ios/pull/6078/commits/bfccf20d8a75ec3fbb012955a221d1cdc411382e

Reply:
Looks like this now, and the code is more elegant to boot.

<img width="373" alt="Screen Shot 2022-02-07 at 2 32 19 PM" src="https://user-images.githubusercontent.com/1595739/152883375-6129a87b-ebe5-4ad0-9bcf-fc616245abac.png">

Reply:
FYI. I checked and confirmed `HStack` will magically layout right to left when the OS passes `.layoutDirection`  `.rightToLeft` into the view (i.e. for Persian, Hebrew, etc.)

Reply:
Cheers for the suggestions @koke @allendav I don't mind if we lose the "or" however, I'm not the biggest fan of the bulleted list. Let's just remove the 'or' if that's okay. Thanks